### PR TITLE
assignment 1

### PR DESCRIPTION
**Pull request by emthoendel**
I'm creating a pull request to compare a blank branch as the base to one with your complete code files so that I can review your code line by line. I have heard of residential firewalls blocking GitHub desktop uploads - so that may have been your issue. If you are unable to build in an exception or troubleshoot that - you should be ok to use the web interface.

### CODE REVIEW

File: assignment01/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<head></head>
+
+<body>
+<!-- James Welu 
+assignment01.htm
+INFO2340.WW
+Thoendel
+12/05/2019
+-->
```
Comment (emthoendel) [inline assignment01/index.html:12]: Super happy to see this comment block!

File: assignment01/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head></head>
```
Comment (emthoendel) [inline assignment01/index.html:4]: this is where I would expect to see the title of the document and any links out to external js files (i.e. jQuery and your custom js).

File: assignment01/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<!-- James Welu 
+assignment01.htm
+INFO2340.WW
+Thoendel
+12/05/2019
+-->
+    <!--
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    -->
+    <script src="js/jquery-3.4.1.min.js"></script>
```
Comment (emthoendel) [inline assignment01/index.html:16]: It's fine for you to locally host jQuery if you want - although a CDN hosted file would be fine as well - often faster.

File: assignment01/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <script src="js/myScript1.js"></script>
+
+<h1>Greetings</h1>
+
+<p>Hello, my name is <span id="first-name">James</span><span id="last-name"> Welu</span>!</p>
```
Comment (emthoendel) [inline assignment01/index.html:21]: The goal of this assignment was to have an html file _exactly_ like the one in figure 1. So there should not be values for your name here, style for the mcc class below or the list items below that. All that content and style should be added using your custom js file.

File: assignment01/js/myScript1.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+$(document).ready();
```
Comment (emthoendel) [inline assignment01/js/myScript1.js:1]: So - you are checking for the document to load here - but you aren't calling or defining a function. The assignment was to create an anonymous function here that sets the name values, appends hobby list items and set the css style for the mcc class. Please let me know if you have questions. We will build on these concepts from here.

**Review by emthoendel — COMMENTED** [review]
You have a good start here, but missed the bulk of the work for the assignment. Our goal is to focus on using scripting languages to adapt our content. Please let me know if you have questions.

File: assignment01/index.html (hunk 2 above)
Comment (jdwelu) [reply assignment01/index.html:4]: Okay, got it! I did not include my scripts within the <head></head> tags on my redux (that was more an attempt at a correct turn in), but I looked it it up and now I understand that is where they should go.

File: assignment01/index.html (hunk 3 above)
Comment (jdwelu) [reply assignment01/index.html:16]: Okay, I will probably just link to the CDN hosted file in the future.

File: assignment01/index.html (hunk 4 above)
Comment (jdwelu) [reply assignment01/index.html:21]: I've got it now. I rewrote the .js and accomplished what was needed to display the page correctly. It's actually included on my last "pull request" I made to try and figure out how things are supposed to be turned in. I should be ready for the next unit.

File: assignment01/js/myScript1.js (hunk 5 above)
Comment (jdwelu) [reply assignment01/js/myScript1.js:1]: I just plain missed the point here. For some reason, I was thinking that the whole point of the assignment was just link the javascript library, and to check if the document was loaded. I just figured that the other tasks were refreshers in html. They are not. I've gone back over it.

**Pull request closed without merging**